extensions/amp-viewer-integration/0.1/messaging.js

```diff
@@ -274,8 +274,8 @@ export class Messaging {
       handler = this.defaultHandler_;
     }
     if (!handler) {
-      throw new Error(
-        'Cannot handle request because handshake is not yet confirmed!');
+      throw new Error('Cannot handle request ' + message.name +
```

Comment:
It's better to use `args`, like this:
```
const error = new Error('Cannot handle ....');
error.args = request.name;
```

This way, we will be able to group errors by the message "Cannot handle..." and get a good total number, but still will be able to find the actual request name if we need to.